Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -643,6 +645,11 @@ def _tear_down(self, timeout=10.0):
     def install_app(self, app_path, env=None):
         # Even after carousel is running, it takes a few seconds for watchOS to allow installs.
         for i in range(self.NUM_INSTALL_RETRIES):
+            # FIXME: remove this workaround after rdar://129789675 has been resolved.
```

Comment:
That is a good point! It would be better if we could just send the XPC message via the command line or script, but I have not found a way to do that yet.

Thanks for reviewing!